app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -262,5 +266,23 @@ def run_generic_operation(operation, ems_ref, parameters = {})
         end
       end
     end
+
+    def run_specific_operation(operation_name, ems_ref, parameters = {})
```

Comment:
can you explain why this method is needed? the `run_generic_operation` method looks very similar